### PR TITLE
Fixes a large amount of spelling and grammatical errors in the Exploration Drones section. Also fixes an angry pine tree quip.

### PR DESCRIPTION
**Pull request by mikederkan**
## About The Pull Request
Title

## Why It's Good For The Game
It bothered me

## Changelog

:cl: Licks-The-Crystal
spellcheck: Corrected a large quantity of spelling, grammatical and phrasing errors with Exploration Drone content.
/:cl:

### CODE REVIEW

**Comment by mikederkan** [comment]
is the conflict resolved oh god i am not good at this

File: code/modules/explorer_drone/exploration_events/resource.dm [hunk 1]
```diff
@@ -168,9 +171,10 @@
 	band_values = list(EXOSCANNER_BAND_PLASMA=1)
 	required_tool = EXODRONE_TOOL_MULTITOOL
 	discovery_log = "Discovered ship fuel storage."
-	description = "You find the ship fuel storage. Unfortunately it's locked with electronic lock."
+	description = "You find the ships fuel storage. Unfortunately, it has an electronic lock."
```
Comment (dragomagol) [inline code/modules/explorer_drone/exploration_events/resource.dm:174]:
```suggestion
	description = "You find the ship's fuel storage. Unfortunately, it has an electronic lock."
```

File: code/modules/explorer_drone/exploration_events/resource.dm [hunk 2]
```diff
@@ -180,9 +184,10 @@
 	required_site_traits = list(EXPLORATION_SITE_SHIP)
 	required_tool = EXODRONE_TOOL_TRANSLATOR
 	discovery_log = "Discovered ship navigation systems."
-	description = "You find the ship navigation systems. With proper tools you can retrieve any data stored here."
+	description = "You find the ships navigation systems encoded in a strange language. You'll be able to use the data with a translator."
```
Comment (dragomagol) [inline code/modules/explorer_drone/exploration_events/resource.dm:187]:
```suggestion
	description = "You find the ship's navigation systems encoded in a strange language. You'll be able to use the data with a translator."
```

File: code/modules/explorer_drone/exploration_events/resource.dm [hunk 3]
```diff
@@ -231,9 +238,10 @@
 	name = "welded locker"
 	required_site_traits = list(EXPLORATION_SITE_SPACE)
 	required_tool = EXODRONE_TOOL_WELDER
-	discovery_log = "You discovered a welded shut locker."
-	description = "You discover a welded shut locker floating through space. What could be inside ?"
-	success_log = "Retrieved bones of unfortunate spaceman from a welded locker."
+	discovery_log = "You discovered a hastily welded locker."
```
Comment (dragomagol) [inline code/modules/explorer_drone/exploration_events/resource.dm:241]: This seems like it should be "Discovered" like the others changed above

File: code/modules/explorer_drone/exploration_site.dm [hunk 4]
```diff
@@ -240,26 +240,26 @@ GLOBAL_LIST_EMPTY(exploration_sites)
 
 /datum/exploration_site/alien_ruins
 	name = "alien ruins"
-	description = "alien ruins on small moon surface."
+	description = "Alien ruins on a small moons surface."
```
Comment (dragomagol) [inline code/modules/explorer_drone/exploration_site.dm:243]:
```suggestion
	description = "Alien ruins on a small moon's surface."
```

**Pull request merged**